src/ceph-daemon/ceph-daemon

```diff
@@ -967,6 +967,19 @@ def command_version():
 
 ##################################
 
+def command_pull():
+    # type: () -> int
+    logger.info('Pulling latest %s...' % args.image)
+    call_throws([container_path, 'pull', args.image])
+    out, err, ret = call_throws([
+        container_path, 'inspect',
+        '--format', '{{.Id}}',
+        args.image])
+    print(out.strip())
+    return 0
```

Comment:
why return `0`?

Reply:
process exit code.  (and because mypy told me to.)